src/modules/tests/test-data/cat-c/_tests_/test-data.cat-c.selector.spec.ts

```diff
@@ -0,0 +1,382 @@
+import { CatCUniqueTypes } from '@dvsa/mes-test-schema/categories/C';
+import {
+  hasSeriousFault,
+  hasDangerousFault,
+  getETAFaultText,
+  getEcoFaultText,
+  getShowMeQuestionOptions,
+} from '../../common/test-data.selector';
+import {
+  getDrivingFaultCount,
+  getManoeuvres,
+  hasManoeuvreBeenCompletedCatBE,
+  areTellMeQuestionsSelected,
+  areTellMeQuestionsCorrect,
+  hasVehicleChecksBeenCompletedCatBE,
```

Comment:
CatBE named imports from cat-c.selector - is this temporary until the selector is reworked?

Reply:
I am not sure if the names should contain the category at all. Some of them do some of them don't..

Reply:
Changed them to cat C for now